dev-infra/format/run-commands-parallel.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Bar} from 'cli-progress';
+import {cpus} from 'os';
+import {exec} from 'shelljs';
+
+const AVAILABLE_THREADS = Math.min(cpus().length - 1, 1);
```

Comment:
Did you mean:

```suggestion
const AVAILABLE_THREADS = Math.max(cpus().length - 1, 1);
```

:grin: